Src/DependencyCollector/DependencyCollector/DependencyCollector.csproj

```diff
@@ -1,4 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\..\..\packages\MicroBuild.Core.0.3.0\build\MicroBuild.Core.props" />
+  <Import Project="$(MicroBuildPluginDirectory)\MicroBuild.Plugins.Signing.1.1.16\build\MicroBuild.Plugins.Signing.props" Condition="Exists('$(MicroBuildPluginDirectory)\MicroBuild.Plugins.Signing.1.1.16\build\MicroBuild.Plugins.Signing.props')" />
```

Comment:
This fix will break as soon as plugin is updated to 1.1.17. We'd need to consider a more permanent fix by then.

Reply:
I believe we would be isolated from this... 1.1.17 is a dependency of MicroBuild.Core 0.3.0
When MicroBuild.Core gets updated, we would also have to fix this.
But i hope to have this problem resolved before then :)